### PR TITLE
[#85zrmkxv1] Adds Alembic Migrations

### PR DESCRIPTION
**Pull request by BattleBaxx**
- Adds Poetry
- Adds Alembic Migrations Init

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Database credentials
```
Comment (SakethRamanKS) [inline .env.example:1]: Add reasonable default values

File: kori/alembic/versions/22ac11d1898b_create_organisation_table.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+# revision identifiers, used by Alembic.
+revision = '22ac11d1898b'
+down_revision = None
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    op.create_table(
+        "organisation",
```
Comment (SakethRamanKS) [inline kori/alembic/versions/22ac11d1898b_create_organisation_table.py]: change to organization everywhere

File: kori/app/core/config.py [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+from dotenv import find_dotenv
+from pydantic import BaseSettings, PostgresDsn, validator
+
+
+class Settings(BaseSettings):
+    class Config:
+        env_file = find_dotenv(usecwd=True)
+        env_file_encoding = "utf-8"
+
+    server_host: str
```
Comment (SakethRamanKS) [inline kori/app/core/config.py]: Capitalize these variables

**Review by SakethRamanKS — COMMENTED** [review]
Phix

**Review by RazCrimson — APPROVED** [review]

**Pull request merged**